common/core/web/utils/package.json

```diff
@@ -6,7 +6,7 @@
   "types:": "./dist/index.d.ts",
   "scripts": {
     "build": "node ../../../../resources/gosh.js ./build.sh",
-    "prepare": "npm run build",
+    "prepare": "npm run build -- -skip-package-install",
```

Comment:
I don't think this will pass the parameter `-skip-package-install` to the `build.sh` script will it?

Reply:
It will.  When running an npm script, anything before the `--` is a parameter on the `npm run`.  Anything after the `--` is a parameter on the script called by `npm run`.

Reply:
Though... there's the question of if `gosh.js` is written properly enough to pass the parameter along.  Let me check that.

Reply:
Based on https://github.com/keymanapp/keyman/blob/master/resources/gosh.js#L32, I'm assuming the answer is yes.

Reply:
Yeah, gosh is designed to pass all the parameters along :) and tested.